docs/how-to-install.md

```diff
@@ -99,7 +99,7 @@ After configuring the Gravitino server, start the Gravitino server by running:
 ./bin/gravitino.sh start
 ```
 
-You can access the Gravitino Web UI by typing <http://localhost:8090> in your browser. or you
+You can access the Gravitino Web UI by typing [http://localhost:8090](http://localhost:8090) in your browser. or you
 can run
```

Comment:
Should line 103 and line 102 be combined into one line?

Reply:
Unnecessary, Markdown will only render empty lines as line breaks.